docs: add a glossary page to describe some common terms (#347)
RetroAchievements/docs: docs/general/glossary.md

---
title: Glossary
description: Breakdown of terms you might see all around the website
---

# Glossary

Below is a list of terms used around the RetroAchievements site described to help explain them!

## Award / Badge

> Awards, also sometimes referred to as badges, are given for completing various tasks.

Reviewer: Awards are currently expressed only as badges, but this will change very likely before the end of the year. Not sure how we can make those two terms not synonymous.

Author: I see them referred to as Awards in events and the main profile page, which is where I got this from. Though I've heard them referred to as badges all the time colloquially.

Reviewer: This should be fine 👍

There are three different types of awards
* Event Awards – Given for completing [events](events.md).
* Game Awards – Given for [completing](#completion) or [mastering](#mastery) game sets.
* Site Awards – Special awards given out for completing certain tasks

## Cheevo

Short for "Achievement".

## Core

> A core, also referred to as a libretro core, is a specific emulator used in RetroArch (or RALibretro) to play a particular game.

Reviewer: Cores are also used in other verified emulators like BizHawk.

For example, you might use the DeSmuMe or MelonDS DS core for Nintendo DS games.

You would manage all of this inside RetroArch and download the cores needed for the games there.
This is why RetroArch is listed as supporting many systems.

## Completion

Completion is the [softcore](#softcore) version of a [mastery](#mastery).

You earn it for collecting every achievement in a game or [set](#set).

## Developer

Around the site, you'll see "developers" and sets "developed by" users of RetroAchievements.
These are almost always users of RetroAchievements developing the achievement set, not the game.

> Whenever you see someone listed as a "Developer" or "Jr. Developer", they are individuals who develop sets for the site.

Reviewer: ```suggestion
Whenever you see someone listed as a "Developer" or "Jr. Developer", they are individuals who develop achievement sets for the site.
```

## Event

> An event is a set of challenges put on by the community to challenge players into doing various tasks for a badge.

Reviewer: ```suggestion
An event is a list of challenges put on by the community to challenge players into doing various tasks for a badge.
```

See [the dedicated page](events.md) for more information and some active events you can hop into!

## Hardcore

"Hardcore" is a mode for achievement collecting that disables certain emulator features to simulate playing as you would on a real console.

Collecting every achievement in a [set](#set) in hardcore mode gives you a [mastery](#mastery) of the game, and a badge for your profile.

See the [FAQ entry](faq.md#what-is-hardcore-mode) for more information about Hardcore/Softcore mode.

## Mastery

This is when you collect every achievement in a game's set in [Hardcore](#hardcore) mode.

Mastering a game awards you a [badge](#award-badge) for your profile.

## RetroPoints

The "white points" you see next to the blue normal points. A tooltip describes them as the points adjusted for rarity.

See the [FAQ entry](faq.md#what-are-the-white-points) for more information.

## Set

A "set" is a collection of achievements for a game.

> Game and set are usually used interchangeably, as many games have one base set, but there are a handful of subsets.

Reviewer: ```suggestion
Game and set are usually used interchangeably, as many games have only one achievement set. A game may also have a handful of subsets.
```

See [what is an achievement set?](how-ra-works.md#what-is-an-achievement-set) for more information.

### Multiset

> Multiset is a tool for subsets that allow you to earn achievements towards multiple subsets at once.

Reviewer: ```suggestion
Multiset is an emulator feature that allows you to earn achievements towards multiple achievement sets for a single game at once.
```

See [the dedicated subsets page](../guidelines/content/subsets.md) for more information.

### Subset

> A subset is an additional set for a game, containing additional challenges or requirements that are beyond the main game's set itself.

Reviewer: ```suggestion
A subset is an additional set for a game, containing additional challenges or requirements that are beyond the requirements of the game's base set.
```

You can see these by going to a game's page and navigating to the other icons above the achievement list.

See [the dedicated subsets page](../guidelines/content/subsets.md) for more information.

## Softcore

"Softcore" is a mode for achievement collecting that allows you to use all of an emulator's features. 

Collecting every achievement in a set, in either hardcore or softcore, gives you a [Completion](#completion) of the game, and a [badge](#award-badge) for your profile.

See the [FAQ entry](faq.md#what-is-hardcore-mode) for more information about Hardcore/Softcore mode.

## Standalone

Typically, all games run on an emulator. However, some games run standalone.

> Currently, there are just Terraria and Final Fantasy XI, but more may be added in the future.

Reviewer: This is very likely to drift/be stale this year. We should mention Terraria and FFXI, but maybe in a way that doesn't require an update once a 3rd standalone is added.

Author: How about

> Typically, all games run on an emulator. However, some games run standalone (e.g. Final Fantasy XI and Terraria)

So this just lists them off as examples and not something that needs updating.

Reviewer: That sounds good 👍

Reviewer: Went ahead and pushed up your proposed change as-written. Thanks!

You'll see these under "Standalone" or "EXE".

Standalone games run as is, without an emulator, and require something else, like a mod, to get you connected to RetroAchievements.

## Ticket

> A ticket is a report of an issue with an achievement. You'll see a button for this on the side of game pages.

Reviewer: ```suggestion
A ticket is a bug report for an achievement.
```
Removed the 2nd sentence - we don't want newer users to think about tickets too much IMO.

---
title: Glossary
description: Breakdown of terms you might see all around the website
---

# Glossary

Below is a list of terms used around the RetroAchievements site described to help explain them!

## Award / Badge

Awards, also sometimes referred to as badges, are given for completing various tasks.

> There are three different types of awards

Reviewer: ```suggestion
There are three different types of awards:
```

* Event Awards – Given for completing [events](events.md).
* Game Awards – Given for [completing](#completion) or [mastering](#mastery) game sets.
* Site Awards – Special awards given out for completing certain tasks

## Cheevo

Short for "Achievement".

## Core

A core, also referred to as a libretro core, is essentially a program that runs through emulators like RetroArch, RALibretro, and BizHawk to play a particular game.

> For example, you might use the DeSmuMe or MelonDS DS core for Nintendo DS games.

Reviewer: ```suggestion
For example, you might use the DeSmuME or melonDS DS core for Nintendo DS games.
```
